packages/components/input/src/input.vue

```diff
@@ -356,6 +377,12 @@ const handleInput = async (event: Event) => {
     value = props.formatter(value)
   }
 
+  if (props.countGraphemes && isFunction(props.countGraphemes)) {
+    const graphemes = props.countGraphemes(value)
+    if (graphemes > Number(maxlength.value)) {
```

Comment:
Is it necessary to judge the type of some `graphemes` here?

Reply:
I think it’s okay not to add it, because `props.countGraphemes(value)` is written in the document and the return value is of type number.

Reply:
ok